includes/Modules/AdSense.php

```diff
@@ -423,6 +446,14 @@ protected function parse_data_response( Data_Request $data, $response ) {
 				return $response;
 			case 'GET:sites':
 				return $response->getSites();
+			case 'GET:ad-blocking-recovery-tag':
+				$this->options->set( self::AD_BLOCKING_RECOVERY_TAG_OPTION, base64_encode( $response->getTag() ) );
+				$this->options->set( self::AD_BLOCKING_RECOVERY_ERROR_PROTECTION_TAG_OPTION, base64_encode( $response->getErrorProtectionCode() ) );
```

Comment:
Hi, I've addressed all other feedback except this one and the "extract a dedicated `Setting`" as I need a few clarification for this. Do you think it will be better if we tackle this two in a follow up issue instead? Cheers.

Reply:
PS: I've filled the [follow up issue](https://github.com/google/site-kit-wp/issues/7156) and assigned it to you as well. Cheers.